app.rb

```diff
@@ -36,9 +36,10 @@ def list_people
   def create_person
     print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
     number = gets.chomp.to_i
```

Comment:
- [ ] for all the methods that create `books, students, teacher, rental` please make sure to separate the code that `requests for input and input validation` like these in the picture
<img width="352" alt="Screenshot 2023-03-13 at 16 08 46" src="https://user-images.githubusercontent.com/61790587/224743361-01888ca1-fa0b-44e6-8a38-9bb97d7f555c.png">
into their own functions (methods) and then use them inside create_student, create_book... etc
that way you will ensure that you're respecting the rule of the single responsibility principle

Reply:
Code has been fixed as requested